docs/qe-available-tasks/sealights/sealights-get-refs.md

```diff
@@ -0,0 +1,144 @@
+# Sealights Get Refs Tekton Task
+
+## Overview
+
+The `sealights-get-refs` Tekton Task is designed to retrieve metadata related to Sealights instrumentation in a CI pipeline. This Task extracts key information, such as the Sealights Build Session ID, the source artifact type, the instrumented container image, and the build name. It does so by processing attestation data associated with a container image using `cosign`.
+
+## Task Description
+
+The Task performs the following steps:
+1. Retrieves the container image associated with the specified component from the provided `SNAPSHOT` parameter.
+2. Downloads the attestation metadata using `cosign`.
+3. Parses the attestation metadata to extract Sealights-related information.
+4. Writes the extracted information to Tekton Task results.
+
+## Parameters
+
+| Name       | Type   | Description                                  |
+|------------|--------|----------------------------------------------|
+| `SNAPSHOT` | string | The JSON string representing the Snapshot under test. |
+
+## Results
+
+| Name                         | Description                                                                     |
+|-------------------------------|---------------------------------------------------------------------------------|
+| `sealights-source-artifact`  | The source code OCI artifact that was instrumented with sealights.             |
+| `sealights-bsid`             | The Build Session ID (BSID) generated during the build for sealights.          |
+| `sealights-container-image`  | The container image used in the Sealights instrumentation process.             |
+| `sealights-build-name`       | The build name generated by sealights instrumentation.                         |
+| `container-image`            | The container image built from the specified Git revision without Sealights instrumentation. |
+
+## Usage Example
+
+Here's an example of how to use the `sealights-get-refs` Task in a Tekton Pipeline:
```

Comment:
Is this meant to run only in the integration pipeline and is it meant to run after test-metadata?

Reply:
I will say integration tests. Let me add it to the docs.